src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Get.AsyncEnumerable.cs

```diff
@@ -166,11 +183,8 @@ public static partial class HttpClientJsonExtensions
                     .ConfigureAwait(false);
                 response.EnsureSuccessStatusCode();
 
-                using Stream readStream = await GetHttpResponseStreamAsync(client, response, false, cancellationToken)
-                    .ConfigureAwait(false);
-
-                await foreach (TValue? value in JsonSerializer.DeserializeAsyncEnumerable<TValue>(
-                    readStream, jsonTypeInfo, cancellationToken).ConfigureAwait(false))
+                await foreach (TValue? value in response.Content.ReadFromJsonAsAsyncEnumerable<TValue>(
```

Comment:
Please revert this change, it is unrelated to the fix at hand and the `GetHttpResponseStreamAsync` uses logic that is subtly different to reading directly from `HttpContent` values.

Reply:
Reverted